root/report/EventSequenceNotInSeries.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+import formatUserDate from '../utility/formatUserDate';
+import {l} from '../static/scripts/common/i18n';
+
+import EventList from './components/EventList';
+import FilterLink from './FilterLink';
+import type {ReportDataT, ReportEventT} from './types';
+
+const EventSequenceNotInSeries = ({
+  $c,
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportEventT>) => (
+  <Layout fullWidth title={l('Events which should be part of series or larger event')}>
+    <h1>{l('Events which should be part of series or larger event')}</h1>
+
+    <ul>
+      <li>
+        {l(`This report lists events where the event name indicates that it may
+        have to be part of a series or a larger event.`)}
```

Comment:
```suggestion
            have to be part of a series or a larger event.`)}
```
(so that it starts at the same column as "This" in the first line)

Reply:
I'm pretty sure that's how I had it at first and eslint complained?

Reply:
Or maybe that was something else :D Changed now.